### ISSUE TITLE
Missing dependencies for openneuro-cli

### ISSUE THREAD

**Issue by chrisgorgo** [OpenNeuroOrg/openneuro#860]
```
login1.ls5(1027)$ node --version
v8.11.4
login1.ls5(1028)$ npm install -g openneuro-cli
/work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/bin/openneuro -> /work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/lib/node_modules/openneuro-cli/src/index.js
+ openneuro-cli@2.1.5
updated 1 package in 4.165s
login1.ls5(1029)$ which openneuro
/work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/bin/openneuro
login1.ls5(1030)$ which node
/work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/bin/node
login1.ls5(1031)$ openneuro
file:///work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/lib/node_modules/openneuro-cli/node_modules/openneuro-client/src/client.js:1
Error: Cannot find module 'apollo-client-preset'
    at Object.Module._extensions..js (module.js:663:10)
login1.ls5(1032)$ npm install -g apollo-client-preset
npm WARN deprecated apollo-client-preset@1.0.8: for the best quick start with Apollo Client, please use apollo-boost instead
npm WARN apollo-cache-inmemory@1.2.9 requires a peer of graphql@0.11.7 || ^0.12.0 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-link@1.2.2 requires a peer of graphql@^0.11.3 || ^0.12.3 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-link-http@1.5.4 requires a peer of graphql@^0.11.0 || ^0.12.0 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-client@2.4.1 requires a peer of graphql@^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN graphql-tag@2.9.2 requires a peer of graphql@^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-link-http-common@0.2.4 requires a peer of graphql@^0.11.0 || ^0.12.0 || ^0.13.0 but none is installed. You must install peer dependencies yourself.

+ apollo-client-preset@1.0.8
added 18 packages in 1.976s
login1.ls5(1033)$ openneuro
file:///work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/lib/node_modules/openneuro-cli/node_modules/openneuro-client/src/client.js:1
Error: Cannot find module 'apollo-link-ws'
    at Object.Module._extensions..js (module.js:663:10)
login1.ls5(1034)$ npm install -g apollo-link-ws
npm WARN apollo-link-ws@1.0.8 requires a peer of subscriptions-transport-ws@^0.9.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-link@1.2.2 requires a peer of graphql@^0.11.3 || ^0.12.3 || ^0.13.0 but none is installed. You must install peer dependencies yourself.

+ apollo-link-ws@1.0.8
added 7 packages in 1.438s
login1.ls5(1035)$ openneuro
/work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/lib/node_modules/openneuro-cli/node_modules/apollo-upload-client/lib/main/index.js:1
Error: Cannot find module 'core-js/modules/web.dom.iterable'
    at Object.<anonymous> (/work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/lib/node_modules/openneuro-cli/node_modules/apollo-upload-client/lib/main/index.js:14:1)
````

Am I doing something wrong?

**Comment by nellh** [comment]
It's broken, looks like b9f671384 introduced a dependency on a package that isn't in package.json. This works for the Lerna environment because the package is present in openneuro-server.

**Comment by chrisgorgo** [comment]
It would be good to add a smoke test of `openneuro` after a fresh independent install in the CI.

**Comment by nellh** [comment]
I'm trying to figure out a way to test this before the dependencies are published. A simple isolated install would test the dependent packages from npm and then after the test succeeded, the new ones would be published and could be wrong still.

We could do something like a test publish to a private npm repository and install from that?

**Comment by nellh** [comment]
This is fixed in openneuro-cli@2.1.11 keeping this issue open to add the test for it.
